pixi.toml

```diff
@@ -88,6 +88,10 @@ sqlalchemy = "2.*"
 pytest-cov = "*"
 pytest-xdist = "*"
 
+[target.unix.dependencies]
```

Comment:
I guess you don't need `memray` in every unix-like installation, but only while developing perhaps.

As such, I'd suggest to create a dev feature/environment and define there a particular platform dependency

```toml
[feature.dev.target.unix.dependencies]
pytest-memray = "*"
memray = "*"
```

Even better, do this in a separate PR.

Reply:
Memray is used within a test case to verify that the caching is truly disabled --> required as a dependency.
Of course, if we do not want this additional overhead, we can simply remove the testcase: 

https://github.com/Quantco/datajudge/blob/f14a54b841ec535be1c70965d8cc873e42491aaa/tests/integration/test_integration.py#L1081-L1112